### DISCUSSION TITLE
Storyboard skin sprites are scaled incorrectly

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#26238] [Q&A] [answered]
noticeable in the tutorial map. in the video i show the following skins in order:

- classic (broken)
- argon (not broken)
- argon pro (not broken)
- triangles (not broken)
- [random custom skin](https://github.com/ppy/osu/files/13795488/-._.WhiteCat.1.0._CK_._-.cyperdark.zip) (broken)

https://github.com/ppy/osu/assets/36758269/1ede4807-b6b9-4450-a259-5ec7411bae99

version: 2023.1229.1-lazer

**Comment by bdach** [comment, marked as answer]
not sure why this isn't an issue. https://github.com/ppy/osu/issues/26239
